examples/plotting/file/categorical_scatter_jitter.py

```diff
@@ -1,19 +1,26 @@
-from bokeh.io import output_file, show
+''' A categorical scatter plot based on GitHub commit history demonstrates
+using a jitter transform.
+
+.. bokeh-example-metadata::
+    :sampledata: :ref:`sampledata_commits`
+    :apis: :func:`~bokeh.plotting.Figure.scatter`
+    :refs: :ref:`userguide_categorical` > :ref:`userguide_categorical_scatters` > > :ref:`userguide_categorical_scatters_jitter`
```

Comment:
Double `> >`, or is this a feature? Though really, and repeating my previous point, isn't there a way to get this out of Sphinx without all the manual labor and repetition? If anything in the doc structure changes, it will be a tedious task to update and make consistent over time.

Reply:
Just a typo. I have already considered this, I would like a "breadcrumbs" directive that can walk up the headings hierarchy in the document and extract this automatically. But there are several questions:
* is it even possible (I suspect, but it will take investigation. I don't know how to get the info needed offhand)
* will it make the build much slower (the build is already quite long)
* how to accommodate more complicated cases, e.g. in at least one place already I have `foo > bar > a, b` to condense things

I want to be clear that my priority is to get a dramatic improvement for users in place for this release. This is the 80% improvement (from literally nothing to a much more context-rich UX for users) that I can get done this week. Diving into research projects right now would risk not being able to finish the user parts and that is not an acceptable risk for me. There are lots of chances for improvements (mostly for us) in the future. 

> If anything in the doc structure changes, it will be a tedious task to update and make consistent over time.

Just noting that the build will at least fail if any reference is wrong, and also that the conventional naming scheme that is mostly used throughout helps greatly.